application/F3DOptionsParser.cxx

```diff
@@ -6,7 +6,7 @@
 #include "F3DSystemTools.h"
 
 #include "cxxopts.hpp"
-#include "json.hpp"
+#include "nlohmann/json.hpp"
```

Comment:
why moving one and not the other ?

Reply:
> why moving one and not the other ?

their header files are at different directories.

```bash
[ruby@fedora fedora-src]$ rpm -ql cxxopts-devel | grep /usr/include/cxxopts.hpp
/usr/include/cxxopts.hpp
[ruby@fedora fedora-src]$ rpm -ql json-devel | grep /usr/include/nlohmann/json.hpp
/usr/include/nlohmann/json.hpp
```

Reply:
Ok, thats fair. Seems also the case on ArchLinux.